music_assistant/constants.py

```diff
@@ -697,20 +698,23 @@ def create_sample_rates_config_entry(
 
 
 # CACHE categories
+class CacheCategory(IntEnum):
```

Comment:
Would it be an idea to have this enum defined in the cache controller module itself ?
For visibility/discoverability I mean